apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2Test.java

```diff
@@ -35,18 +35,8 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.hl7.fhir.instance.model.api.IBaseDatatype;
-import org.hl7.fhir.r4.model.Address;
-import org.hl7.fhir.r4.model.BooleanType;
-import org.hl7.fhir.r4.model.Coding;
-import org.hl7.fhir.r4.model.DateTimeType;
-import org.hl7.fhir.r4.model.DateType;
+import org.hl7.fhir.r4.model.*;
```

Comment:
I can revert this change if needed, something in Intellij keeps insisting on changing it.

Reply:
Found the setting that was doing this, it's fixed now.